pkg/lib/synthesize/non_inferrable_csv.go

```diff
@@ -0,0 +1,72 @@
+package synthesize
+
+import (
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	operatorApiVersion "github.com/operator-framework/api/pkg/lib/version"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+)
+
+type NonInferableCSV struct {
+	Name           string                             `json:"name"`
+	Version        operatorApiVersion.OperatorVersion `json:"version"`
```

Comment:
nit: looks like the import name is colliding with the variable name ie we have "operator" and "version" in both the import name and the variable name, which is considered unidiomatic. Since it looks like this package is only imported and used here I suggest renaming the import from `operatorApiVersion` to something like `apiver` which I don't think loses any meaning around the `Version` variable semantically.